packages/loader/container-loader/src/container.ts

```diff
@@ -1563,6 +1565,12 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         return deltaManager;
     }
 
+    private registerSubmitOpEvent() {
```

Comment:
It would be great to replace same code in createDeltaManager() and call this function instead

Reply:
Removed this event.